git.md

```diff
@@ -0,0 +1,73 @@
+# Git Best Practices
+
+CFPB uses the Git version control system for tracking changes to our code and documentation.
+GitHub.com is used to share and manage our Git repositories.
+
+## Signing commits using GPG
+
+![GitHub verified commit](img/github-verified.png)
+
+GPG keys are a way to sign and verify work from trusted collaborators.
+You can generate a GPG key and add the public key to your account by following [GitHub's guide](https://help.github.com/articles/signing-commits-with-gpg/).
+
+### Why is commit signing important?
+
+Anyone can make a commit to a codebase and attribute it to you: `git commit --author='Jane Doe <jane@doe.com>' -m 'Really evil malicious change'`.
+It will show up in the project's [log](https://github.com/cfpb/capital-framework/commits/master) with your name and smiling face (assuming your GitHub email address is `jane@doe.com`).
+
+To protect yourself and prove to your peers that you are the author of a given commit, you can sign the commit using your GPG key.
+Assuming your development machine is not compromised and you generated your key with a strong passphrase, your colleagues can rest assured any commits with a "verified" badge next to them were written by you.
+For more info read Mike Gerwitz's "[A Git Horror Story: Repository Integrity With Signed Commits](https://mikegerwitz.com/papers/git-horror-story)".
+
+## Writing good commit messages
+
+CFPB's [dotfiles](https://github.com/cfpb/dotfiles) include a git commit message template that encourages some [best practices](https://chris.beams.io/posts/git-commit/).
+
+### How to use the message template
+
+Download the git commit message template to your home directory:
+
+```sh
+curl https://raw.githubusercontent.com/cfpb/dotfiles/master/.gitmessage -o ~/.gitmessage
+```
+
+Tell git to use it:
+
+```
+git config --global commit.template ~/.gitmessage
+```
+
+Whenever you create a new commit, use `git commit -a` (or just `git ca` if you're using our [aliases](https://github.com/cfpb/dotfiles/blob/9342ffb879c3134a9b249fea7c50486679f79a8a/.gitconfig#L35)).
```

Comment:
You could leave out the `-a` flag and mention of the `ca` alias. It's not really relevant to making use of the commit message template.